app/views/trails/show.json.jbuilder

```diff
@@ -8,4 +8,11 @@ json.distance @trail.distance
 json.duration @trail.duration
 json.food_supply @trail.food_supply
 json.warning @trail.warning
-json.photo_urls @trail.photo_urls
\ No newline at end of file
+json.photo_urls @trail.photo_urls
+json.start_longitude @trail.start_longitude
```

Comment:
these should be 
json.start_coordinates @trail.start_coordinates
json.end_coordinates @trail.end_coordinates